lib/phoenix_live_view.ex

```diff
@@ -875,28 +875,41 @@ defmodule Phoenix.LiveView do
   The update function receives the current key's value and
   returns the updated value. Raises if the key does not exist.
 
+  The update function may also be of arity 2, in which case it receives
+  the current key's value as the first argument and the current assigns
```

Comment:
I reused the grammar from the previous paragraph in "the current key's value", but they may read better as "the key's current value". Any interesting in updating them at this opportunity?

Reply:
> [selected|provided

I feel this is implied by the function's context, but happy to follow any recommendation from the maintainers.